music21/harmony.py

```diff
@@ -2492,6 +2493,33 @@ def transpose(self: NCT, _value, *, inPlace=False) -> NCT | None:
 
 # ------------------------------------------------------------------------------
 
+class FiguredBassIndication(Harmony):
+    isFigure = True
+    tstamp = 1
```

Comment:
If these are class attributes, won't each instance contaminate the the others when attempting to set them? Also, can you explain what tstamp is needed for? (Can we just use `.duration` and `writeAsChord=False`?)